Update plips.md: be more precise about mentioning teams to submit a PLIP (#1972)
plone/documentation: docs/contributing/core/plips.md

If you need help at any point in this process, you can either `@` the team or personally contact a member of the designated team.

> ## Team approves proposal

Reviewer: Let's add an anchor.

```suggestion

(team-approves-proposal-label)=

## Team approves proposal
```

After incorporating feedback to your proposal, you can request a final review and approval for inclusion in Plone.

> You can do this by mentioning either one of the designated teams (@plone/Volto or @plone/ClassicUI or meantion @plone/framework-team if you are not sure if your PLIP is suited for a designated team.

Reviewer: ```suggestion
You can do this by mentioning one of the appropriate {ref}`designated-teams-label`, with `@plone/framework-team` as the default if you're not certain.
```

> Every PLIP must be approved by the designated team. Once your PLIP has been approved by one of the three teams, you can start implementing your PLIP.

Reviewer: This contradicts a later line at https://github.com/plone/documentation/pull/1972/files#diff-a26160e65b3eaac12980b0e79dbd62bda9260754c0be27cf10424261c219e2e5R117. Y'all need to make a decision on when a developer may start work on their PLIP. I'd update or delete the later line, as it subverts the process.

One sentence per line, please.

```suggestion
Every PLIP must be approved by the designated team for work to be merged.

Although work may be started at any time, the PLIP may be revised many times or even rejected.
Set expectations accordingly before starting work on the PLIP.
In general, it's a good idea to wait for approval of the PLIP before starting work, but there are exceptions, such as creating a proof of concept to demonstrate what the PLIP would be designed to do, or you might need it for your own project anyway.

```

Reviewer: @tisto please take a look at this comment. There's a problem that I don't know how to resolve. Thank you!

Reviewer: @tisto or anyone else who is deeply familiar with the PLIP process, can I get clarification on this contradiction?

Reviewer 2: @stevepiercy Personally, I think it's dumb to tell people they can't start working on something they're interested in. That's what branches are for, and we can't actually enforce it. (And I don't think we should...sometimes a proof of concept is necessary to demonstrate the usefulness of an idea.)

The main thing we're trying to communicate here is that people working on or reading the PLIP shouldn't assume it's desired in Plone core until it's been reviewed and approved by a team. So don't get upset if you put a lot of work into something prematurely. And don't assume that something is ready to work on just because someone submitted a PLIP that hasn't been reviewed.

Reviewer: @davisagli I think we're in agreement. Since @tisto hasn't replied, I'll make the change to agree with https://github.com/plone/documentation/pull/1972/files#diff-a26160e65b3eaac12980b0e79dbd62bda9260754c0be27cf10424261c219e2e5R117 and merge this.

Reviewer: @davisagli would you please review the revised statement in my suggestion at https://github.com/plone/documentation/pull/1972/files#r2354480231 ? I'd love to wrap up this PR. Thank you!

Reviewer 2: @stevepiercy that looks good to me
